app/templates/components/forms/wizard/sessions-speakers-step.hbs

```diff
@@ -1,24 +1,6 @@
 <form class="ui form {{if this.isLoading 'loading'}}" autocomplete="off" {{action 'move' 'forward' on='submit' preventDefault=true}}>
 
-  <div class="ui centered grid">
-    <div class="column">
-      <div class="ui {{if this.data.event.isSessionsSpeakersEnabled 'basic'}} segment">
-        <div class="center aligned text">
-          <div class="field">
-            <div class="ui slider checkbox">
-              <Input @type="checkbox" @checked={{this.data.event.isSessionsSpeakersEnabled}} />
-              <label class="weight-300" style="font-size: large">
-                {{if this.data.event.isSessionsSpeakersEnabled (t 'Turn off') (t 'Turn on')}}
-                {{t 'Sessions & Speakers'}}
-              </label>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-
-  {{#if this.data.event.isSessionsSpeakersEnabled}}
```

Comment:
Removing this doesn't help as now this makes it compulsory to add a session. The switch is needed & should work & should not give any error when toggled "off"

Reply:
Should i enable all the forms (ticket, sponsors, sessions) by default?? Is that the ideal behaviour?

Reply:
No